### PR TITLE
iphone 11

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2019-10-20-iPhone-11.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# iPhone 11
+
+I've now spent one month using iPhone 11 as my daily driver.
+
+It is a big step up from my previous 250 EUR budget Android phone and my 5 year old iPhone 5S. This article should be seen in that light. I hadn't try a modern iOS device as my daily driver before so a lot of things that might be obvious to people using an iPhone X are not to me and I'm sure some things I point out were there in previous iPhones as well.
```
Comment (MsSlacker) [inline _posts/2019-10-20-iPhone-11.md]: I hadn't try -> I have not tried

Sentence is a run on, spilt it up

File: _posts/2019-10-20-iPhone-11.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Pros
+
+- Great Camera (even in low light conditions)
+With both my Android phone and the 5S I had to use my wife's phone or my reflex to be able to get a non grainy picture on any light condition that was less than perfect. With iPhone 11 I don't need to worry, even at night your pictures will be more than acceptable.
```
Comment (MsSlacker) [inline _posts/2019-10-20-iPhone-11.md]: Windows Phone lol Carl Zeiss ftw

on - > in any light condition

your - > the (is is a personal review)

File: _posts/2019-10-20-iPhone-11.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+More than one full day with medium use. I don't worry at all if I forget to charge it one night since my use is not heavy.
+
+- Way better apps than on Android (with a few exceptions, see "Cons")
+The software quality divide is massive. On top of the fact that a lot of great iOS apps are simply not available on Android.
+
+- Hardware Specs
+Needless to say the sheer difference in speed and responsiveness between the iPhone 11 and my previous phones is immense. There were times when my Android phone would freeze for seconds to render some heavy application. Mind that here I'm not talking about heavy games, but normal applications such as Google Music or Waze.
+
+- iOS
+The animations and scrolling on iOS is unmatched on Android. Some is due to the device itself as for the previous point, but a lot of it is just the OS. I have no idea how Google still can't do scrolling content right after all of this years, but the difference is astonishing.
```
Comment (MsSlacker) [inline _posts/2019-10-20-iPhone-11.md]: this - > these

File: _posts/2019-10-20-iPhone-11.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+## Cons
+
+- Face ID is terrible
+It only works on perfect light conditions and even then only 70% of the times. Considering that it is a version 2 since version 1 was last year with the iPhone X it is totally unacceptable. Pretty much at the same level that Touch ID was with the iPhone 5S. Biggest disappointment.
```
Comment (MsSlacker) [inline _posts/2019-10-20-iPhone-11.md]: on - > in

time is not countable, lose the s

put since version 1 (...) in (

Comment (MsSlacker) [reply _posts/2019-10-20-iPhone-11.md]: Windows Hello works better

File: _posts/2019-10-20-iPhone-11.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+- Quick Actions Are Too Slow To Activate
+This might be something that not a lot of people notice, but on Android if you want to turn Wi-Fi on and off, enable torch, turn on the hotspot or data roaming you can do it with one swipe and one tap. Control center on iOS is only limited to 5 actions and those actions are way less than what Android makes available. You can still access additional actions with one more tap, but for actions that are repeated often this takes a lot of time if you sum it up over the months and years. I feel like the screen real estate is there, why not using it effectively?
+
+
+
+All in all I spend 90% of the time on my Mac and when I'm my iPhone I'm often bored with nothing to do. I've decided not to participate in social media, mobile games and the like so there is not much left. The only few productive applications I'm currently using on iOS let me review pull request (GitHawk) and write notes (Teria). There are more that I use sparingly, but these are the ones I tend to use on a daily basis. Whenever I have more than 20 minutes of downtime I go to my Mac to actually do something.
```
Comment (MsSlacker) [inline _posts/2019-10-20-iPhone-11.md]: like, so

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-10-20-iPhone-11.md (hunk 1 above)
Comment (valeIT) [reply _posts/2019-10-20-iPhone-11.md]: Well now I have though

File: _posts/2019-10-20-iPhone-11.md (hunk 2 above)
Comment (valeIT) [reply _posts/2019-10-20-iPhone-11.md]: sticking with "you"

File: _posts/2019-10-20-iPhone-11.md (hunk 4 above)
Comment (valeIT) [reply _posts/2019-10-20-iPhone-11.md]: on - > in
Consistent in my wrong ways I see..

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**